### PR TITLE
MAYA-108691 - USD: undoing transform will not update manipulator

### PR DESCRIPTION
**Pull request by seando-adsk**
#### MAYA-108691 - USD: undoing transform will not update manipulator
* Send UFE Transform3d and AttributeValueChanged in response to USD "prim property path" notif.
* Added new test to verify these notifs.

### CODE REVIEW

File: lib/mayaUsd/ufe/StagesSubject.cpp [hunk 1]
```diff
@@ -205,11 +205,32 @@ void StagesSubject::stageChanged(
...
+                if (!InTransform3dChange::inTransform3dChange()) {
+                    Ufe::Transform3d::notify(ufePath);
+                }
+#ifdef UFE_V2_FEATURES_AVAILABLE
+                if (!inAttributeChangedNotificationGuard()) {
+                    Ufe::AttributeValueChanged vc(ufePath, changedPath.GetName());
+                    Ufe::Attributes::notify(vc);
+                }
+#endif
+            }
```
Comment (seando-adsk) [inline lib/mayaUsd/ufe/StagesSubject.cpp:226]: As we discussed, make sure to send extra UFE notifs for USD prim property path notifs on xformop.

File: test/lib/ufe/testComboCmd.py [hunk 2]
```diff
@@ -812,6 +841,67 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 2, 'testPrimPropertyPathNotifs only available in UFE v2 or greater.')
+    def testPrimPropertyPathNotifs(self):
```
Comment (seando-adsk) [inline test/lib/ufe/testComboCmd.py:845]: New test to verify that these new are received.

File: lib/mayaUsd/ufe/StagesSubject.cpp [hunk 3]
```diff
@@ -205,11 +205,32 @@ void StagesSubject::stageChanged(
...
-        // here with a resync path. However the changedPath is not a prim path, so we
-        // don't care about it. In those cases, the changePath will contain something like:
-        //   "/<prim>.visibility"
-        //   "/<prim>.xformOp:translate"
+        if (changedPath.IsPrimPropertyPath()) {
+            // Special case to detect when an xformop is added or removed from a prim.
+            // We need to send some notifs so Maya can update (such as on undo
+            // to move the transform manipulator back to original position).
+            const TfToken nameToken = changedPath.GetNameToken();
+            auto blah = nameToken.GetString();
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/StagesSubject.cpp:213]: That's the technical use of blah, right?

**Review by ppt-adsk — COMMENTED** [review]
Thanks for the test!  Just that nit-pick debugging statement you can remove in StagesSubject.cpp :)

**Review by ppt-adsk — APPROVED** [review]

**Review by ppt-adsk — APPROVED** [review]

**Pull request merged**